src/g_hslider.c

```diff
@@ -36,7 +36,7 @@ static void hslider_draw_update(t_gobj *client, t_glist *glist)
     t_hslider *x = (t_hslider *)client;
     if (glist_isvisible(glist))
     {
-        int r = text_xpix(&x->x_gui.x_obj, glist) + ((x->x_val + 50)/100);
+        int r = text_xpix(&x->x_gui.x_obj, glist) + ((x->x_pos + 50)/100) * IEMGUI_ZOOM(x);
```

Comment:
the `((x->x_pos + 50)/100) * IEMGUI_ZOOM(x)` should probably go into a (static) helper-function or at least a macro. it's repeated over and over and over.

Reply:
done (and some others) by macros.
see https://github.com/pure-data/pure-data/pull/933/commits/b39bfd260d100b3e06d9a77e11be34c202a4e688